akka-persistence-typed/src/main/scala/akka/persistence/typed/EventRejectedException.scala

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.typed
+
+/**
+ * Thrown if a journal rejects an event e.g. due to a serialization error.
+ */
+final case class EventRejectedException(persistenceId: String, sequenceNr: Long, cause: Throwable)
```

Comment:
Making it a case class will make it close to impossible to evolve while keeping bincomp